populate/src/main/java/bio/terra/pearl/populate/service/EnrolleePopulator.java

```diff
@@ -613,6 +609,40 @@ public void autoCompleteSurvey(ParticipantTask task, Enrollee enrollee, Particip
         }
     }
 
+    private AnswerPopDto createAnswerFromQuestionDef(SurveyQuestionDefinition questionDef) {
+        AnswerPopDto answer = AnswerPopDto.builder()
+                .questionStableId(questionDef.getQuestionStableId())
+                .build();
+
+        if (!StringUtils.isBlank(questionDef.getChoices())) {
+            answer.setStringValue(parseAndSelectRandomChoice(questionDef.getChoices(), "blah"));
+        } else {
+            answer.setStringValue("blah");
```

Comment:
might as well make the text input random too. 
```suggestion
            answer.setStringValue(RandomStringUtils.randomAlphabetic(RandomUtils.nextInt(5,30));
```